source/AI.h

```diff
@@ -179,6 +180,8 @@ template <class Type>
 		// HARVEST is related to MINE and is for picking up flotsam after
 		// ATTACK.
 		static const int HARVEST = 0x003;
+		static const int TRAVEL_TO = 0x004;
+		static const int LAND_ON = 0x005;
```

Comment:
I'm not sure what you mean by this? The way I understand the code (which isn't my own), these are just orders that the AI can issue, linked to otherwise unused integer values.

Reply:
Nvm then i thought the integers represented keys